apps/server/src/api-data/excel/excel.controller.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * This module encapsulates logic related to
+ * Google Sheets
+ */
+
+import { Request, Response } from 'express';
+import { extname, join } from 'path';
+import { existsSync, renameSync } from 'fs';
+import { generateId } from 'ontime-utils';
+import { uploadsFolderPath } from '../../setup/index.js';
+import { getExcelWorksheets, handleMaybeExcel } from '../../utils/parser.js';
+
+export async function postExcel(req: Request, res: Response) {
```

Comment:
I dont really like the idea of having controllers accessing file system. although I know that we have them, we shouldnt make this worse

lets create a excel.service.ts file and push some of the code to do with filesystem there.
Also, I think we have plenty of usages for existSync and renameSync that we could start cleaning up, i reckon we may have reusable functions that do this, or should do if not